server/app/views/api/v1/profiles/_profile.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Can this whole file be removed - is it unused?

Reply:
removed file